crates/stdx/Cargo.toml

```
@@ -23,7 +23,7 @@ itertools.workspace = true
 
 [target.'cfg(windows)'.dependencies]
 miow = "0.6.0"
-windows-sys = { version = "0.52", features = ["Win32_Foundation"] }
+windows-sys = { version = "0.59", features = ["Win32_Foundation"] }
```

```suggestion
windows-sys =  "0.59"
```

should be enough, right?

---

It looks like stdx actually uses something from the `Win32_Foundation` feature and since no other features are being used this would be required.